### PR TITLE
The Ember Times No. 186 - August 16th 2021

### PR DESCRIPTION
**Pull request by amyrlam**
👋 New to The Ember Times? If you want to claim a topic to write about, please comment here on the PR or chat with us in [#support-ember-times](https://discordapp.com/channels/480462759797063690/485450546887786506) on the [Ember Discord](https://discord.gg/emberjs)! 

***

### Addon/App releases

- [x] v3 of ember-responsive-images https://twitter.com/simonihmig/status/1418592938456752128 from https://discord.com/channels/480462759797063690/480499624663056390/868153168096288779 (:lock: @fozy81)
- [ ] ember-user-activity https://twitter.com/AxleHellfire/status/1424825542767874052
- [ ] ember-element-query https://twitter.com/lolmaus_en/status/1424083417591914499
- [ ] ember-engines-router-service https://twitter.com/MVillander/status/1423659543427522560
- [ ] ember-scroll-modifier https://twitter.com/AxleHellfire/status/1423379276104011778

### Articles

 - [x] Chris Manson coding on Twitch https://twitter.com/real_ate/status/1420662570701230080 (🔒 @amyrlam  )
 - [x] Glimmer Apollo https://glimmer-apollo.com/ Jose shared on Discord here if questions https://discord.com/channels/480462759797063690/485450546887786506/871896266093781052 / https://twitter.com/josemarluedke/status/1410295473072611328 (🔒 @amyrlam  )
 - [x] I just wrote on how to avoid did-insert/did-update/will-destroy in #EmberJS by using custom modifiers and resources. https://twitter.com/nullvoxpopuli/status/1421128258427490311 and Preston shared on Discord here if questions https://discord.com/channels/480462759797063690/485450546887786506/872272494843789402 (🔒 @chrisrng )
  - (1) https://github.com/NullVoxPopuli/ember-resources
  - (2) https://github.com/NullVoxPopuli/ember-array-map-resource
  - (3) https://github.com/NullVoxPopuli/ember-data-resources

## 🤔  Questions?

See #support-ember-times on Discord or CONTRIBUTING.

## ✅  Publishing TODOs:

- [x] Check Netlify / localhost preview, to ensure display e.g. buttons on emberjs.com/blog is good
- [x] Crosspost on dev.to: delete any alex ignore's, manually redo Readers Question picture, add Canonical URL 
- [ ] Set up tweet for Monday morning on tweetdeck
- [ ] Post $natural for #news-and-announcements in Discord https://github.com/ember-learn/ember-blog/blob/main/CONTRIBUTING.md#publishing-an-issue-of-the-ember-times
- [x] Shoutout the real MVP's, our writers!

### CODE REVIEW

**Comment by fozy81** [comment]
I've claimed the ember-responsive image addon. Hope that's okay.

**Comment by amyrlam** [comment]
@fozy81 sounds good, that way of claiming is perfect, thanks!

just to xpost from discord, thinking maybe will send this issue out later than friday. i'm pretty sick and going to the doc later. let's see how it goes

**Comment by fozy81** [comment]
Take care @amyrlam don't worry about Ember Times, look after yourself.

File: content/the-ember-times-issue-186.md [hunk 1]
```diff
@@ -11,15 +11,16 @@ tags:
...
+Optimising image size and formats to improve performance can get complicated, but the [`ember-responsive-image`](https://github.com/kaliber5/ember-responsive-image) addon has your back! Now you have even more reason to take advantage of its abilities following the annoucement by [Simon Ihmig (@simonihmig)](https://github.com/simonihmig) of [version 3](https://github.com/kaliber5/ember-responsive-image/releases/tag/v3.0.0). This release provides support for Cloudinary and imgix image CDNs, further enriching a full set of functionality for re-sizing images and optimizing image loading times across all browsers. Go check it out.
```
Comment (amyrlam) [inline content/the-ember-times-issue-186.md]: @fozy81 looks good i made a small copy edit. thank you!

**Pull request merged**